config/locales/pt-BR.yml

```diff
@@ -140,30 +145,30 @@ pt-BR:
         uptime: Uptime
       header:
         dashboard: Painel de Controle
-        edit_profile:
-        settings:
+        edit_profile: Editar Perfil
+        settings: Configurações da Conta
         search_gem_html: Buscar Gems&hellip;
         sign_in: Fazer Login
         sign_out: Sair
         sign_up: Cadastrar
   mailer:
-    confirm_your_email:
-    confirmation_subject:
-    link_expiration_explanation_html:
+    confirm_your_email: Por favor, confirme seu endereço de email através do link que enviamos para o seu endereço de email.
+    confirmation_subject: Por favor, confirme seu endereço de email para se cadastrar no RubyGems.org
```

Comment:
Is this supposed to be `con RubyGems.org`? Speaking only on basis of google translate.

Reply:
nm. no seems correct.

Reply:
In Portuguese, "no" means "in/on something", we could have many variations of that, such as "em", which basically means the same thing.